content/english/java-basics/activity-1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Print Statements and Comments"
+description: "Introduce println() methods and comments in Java."
+date: 2020-07-08T00:00:00Z
+---
+
+### 1. Let the Program Speak! 
+
+Let’s try to print `Hello World` out by typing `System.out.print("Hello World");` 
+
+As you might have guessed, the program prints out whatever you typed in the parenthesis. And, each line of code ends with `;`.
```

Comment:
This might not be necessarily intuitive so "As you might have guessed" might not be the best word choice.